trieste/utils/objectives.py

```diff
@@ -29,6 +29,19 @@
 from ..type import TensorType
 
 
+def branin_internals(x: TensorType, scale: TensorType, translate: TensorType) -> TensorType:
```

Comment:
Mark as internal? Otherwise, it should have a docstring.

```suggestion
def _branin_internals(x: TensorType, scale: TensorType, translate: TensorType) -> TensorType:
```